File: chat-uikit/src/main/java/io/agora/chat/uikit/chat/interfaces/OnPeerTypingListener.java
package io.agora.chat.uikit.chat.interfaces;

public interface OnPeerTypingListener {
    /**
     * Used to monitor peer's data events
     * @param action Input event TypingBegin is start TypingEnd is finish
     */
    default void onPeerTyping(String action){}
}

File: ease-im-kit/src/main/kotlin/io/agora/chat/uikit/feature/chat/interfaces/OnPeerTypingListener.kt
package io.agora.chat.uikit.feature.chat.interfaces

interface OnPeerTypingListener {
    /**
     * Used to monitor peer's data events
     * @param action Input event TypingBegin is start TypingEnd is finish
     */
    fun onPeerTyping(action: String?) {}
}